plinth/modules/firewall/templates/firewall.html

```diff
@@ -40,48 +40,54 @@
 
     <div class="row">
       <div class="col-lg-8">
-        <table class='table table-autowidth table-striped'>
+        <table class='table table-autowidth'>
           <thead>
             <th>{% trans "Service/Port" %}</th>
             <th>{% trans "Status" %}</th>
           </thead>
 
           <tbody>
             {% for service in services|dictsort:"name" %}
-              {% if service.ports %}
-                <tr>
-                  <td><strong>{{ service.name }}</strong></td>
-                  <td>
-                    {% if service.is_enabled %}
-                      <span class='label label-success'>
-                        {% trans "Enabled" %}</span>
-                    {% else %}
-                      <span class='label label-warning'>
-                        {% trans "Disabled" %}</span>
-                    {% endif %}
-                  </td>
-                </tr>
-
-                {% for port in service.ports %}
+              {% if service.is_enabled %}
```

Comment:
My original idea was not to hide services that are disabled, but services that had never been setup.